internal/pkg/cli/job_init.go

```diff
@@ -194,6 +194,9 @@ func (o *initJobOpts) Execute() error {
 		return err
 	}
 	o.platform = platform
+	if o.platform != nil {
+		log.Warningf("Your architecture type is currently unsupported. Setting platform %s instead.\nSee 'platform' field in your manifest.\n", dockerengine.DockerBuildPlatform(dockerengine.LinuxOS, dockerengine.Amd64Arch))
```

Comment:
nit: maybe
```suggestion
		log.Warningf("Your architecture type is currently unsupported. Setting 'platform' to %s instead in your manifest.\n", dockerengine.DockerBuildPlatform(dockerengine.LinuxOS, dockerengine.Amd64Arch))
```

Reply:
Okay, but I'll leave the wording as is in `svc_init` because of the platform field not being surfaced in RDWS.